test/Jamfile

```diff
@@ -68,7 +74,7 @@ local requirements =
         <target-os>linux:<define>_XOPEN_SOURCE=600
         <target-os>linux:<define>_GNU_SOURCE=1
         <target-os>windows:<define>_WIN32_WINNT=0x0601
-        <include>../include
+        <define>BOOST_ASIO_SEPARATE_COMPILATION
```

Comment:
AFAIK Boost.MySQL does not need this macro. It's true we use it throughout the tests to speed up compilation, but it's not a requirement (actually I should define it out when building in header only mode).